pkg/convert/convert.go

```diff
@@ -8,11 +8,49 @@ func convertToMessageFormatSingular(message string) string {
 		return ""
 	}
 
+	message = escapeSpecialChars(message)
+	message = escapePipe(message)
+
 	return "{" + message + "}"
 }
 
+// escapeSpecialChars escapes special characters in a string.
+func escapeSpecialChars(message string) string {
+	if strings.Contains(message, "{") ||
+		strings.Contains(message, "}") ||
+		strings.Contains(message, "\\") {
+		r := strings.NewReplacer("{", "\\{", "}", "\\}", "\\", "\\\\")
+		message = r.Replace(message)
```

Comment:
Simply return it if there are no modifications.

```suggestion
		return r.Replace(message)
```

Reply:
Will adjust. ;)